.github/PULL_REQUEST_TEMPLATE/main-template.md

```diff
@@ -0,0 +1,6 @@
+Tag name (The PR title should also include the tag name):
+Originator(s):
+
+List all `develop` PR numbers included in this PR and the title of each:
```

Comment:
Should this say `development` here instead of `develop`?

Reply:
good call - updated